tests/samples/expected/ocaml.ml

```diff
@@ -596,6 +597,19 @@ let _ =
     x
   )
 
+let _ =
+  let foo = fun x ->
+      x
+  in
+  foo
+
+let _ =
+  let foo = function
+    true -> false
```

Comment:
Ugh. Let's add the preceding `|` . Probably not needed for v0.1 (we'll have the same story as with list terminators, I imagine: we only want the `|` for multi-line only, and remove it for single line. Probably.

(this is definitely not a comment to take into account for this PR.)